client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-feature-previews/jetpack-preview-pane.tsx

```diff
@@ -43,14 +31,23 @@ export function JetpackPreviewPane( {
 
 	const { selectedSiteFeature, setSelectedSiteFeature } = useContext( SitesDashboardContext );
 
+	useEffect( () => {
+		if ( selectedSiteFeature === undefined ) {
+			setSelectedSiteFeature( 'jetpack_boost' );
```

Comment:
If you use a string for this rather than a constant, you will have to replace it everywhere if the default feature changes. So, let's use a constant.

Reply:
Yes, the idea is to have a centralized place to add a list of feature IDs. It goes in my next PR, where I create a `feature` folder. The file will have all the IDs for the features, Jetpack, or other feature families.

Reply:
Is this idea documented anywhere?

Reply:
In the foundation concept and discussed on P2 posts, comments, and meetings. The Sites Dashboard foundation design is to be open, scalable,  and to be able to support any kind of feature families, that means that each feature product will have it's own ID. And also, for each feature family we will have a folder with all the Preview Panes and feature tabs and data configuration.